build.sbt

```diff
@@ -431,8 +472,29 @@ def http4sProject(name: String) = Project(name, file(name))
     initCommands()
   )
 
+def http4sCrossProject(name: String, crossType: CrossType) = CrossProject(name, file(name), crossType, JSPlatform, JVMPlatform)
+  .settings(commonSettings)
+  .settings(
+    moduleName := s"http4s-$name",
+    testOptions in Test += Tests.Argument(TestFrameworks.Specs2,"showtimes", "failtrace"),
+    initCommands()
+  ).jsSettings(
+    // Ignore, tut is not supported in scala.js
+    sources in (Compile,doc) := Seq.empty,
+    // This is needed to support the  TLS compiler and scala.js at the same time
+    // Remove the dependency on the scalajs-compiler
```

Comment:
I'm confused by what's going on here.  Note that we reverted use of the TLS compiler.

Reply:
Oh really, Are we on LBS now? If so I can drop this

Reply:
Yes.  We weren't using any advanced features, and the binary compatibility isn't quite yet as we thought it would be when we adopted it.  I'm interested in going back at some point, but for now, it's LBS.